src/tests/Common/Platform/platformdefines.h

```diff
@@ -177,7 +177,6 @@ error_t TP_itow_s(int num, LPWSTR buffer, size_t sizeInCharacters, int radix);
 error_t TP_itoa_s(int num, LPSTR buffer, size_t sizeInCharacters, int radix);
 LPWSTR TP_sstr(LPWSTR str, LPWSTR searchStr);
 LPSTR  HackyConvertToSTR(LPWSTR pwszInput);
-DWORD TP_CreateThread(THREAD_ID* tThread, LPTHREAD_START_ROUTINE worker,  LPVOID lpParameter);
 void TP_JoinThread(THREAD_ID tThread);
```

Comment:
Delete `TP_JoinThread` as well? It you cannot create thread, you cannot join them either.